packages/web-components/src/badge/badge.options.ts

```diff
@@ -1,19 +0,0 @@
-/**
- * A Badge can be filled, outline, ghost, inverted
- */
-export type BadgeAppearance = 'filled' | 'ghost' | 'outline' | 'tint';
-
-/**
- * A Badge can be one of preset colors
- */
-export type BadgeColor = 'brand' | 'danger' | 'important' | 'informative' | 'severe' | 'subtle' | 'success' | 'warning';
-
-/**
- * A Badge can be square, circular or rounded.
- */
-export type BadgeShape = 'circular' | 'rounded' | 'square';
-
-/**
- * A Badge can be on of several preset sizes.
- */
-export type BadgeSize = 'tiny' | 'extra-small' | 'small' | 'medium' | 'large' | 'extra-large';
```

Comment:
If we can take this to an issue or alternate PR, I'm down to take the rest of this.

Reply:
Moved, thanks

Reply:
#5